src/db/migrations/20230623152940_add_onerep_scans_table.js

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export function up (knex) {
+  return knex.schema
+    .createTable('onerep_scans', table => {
+      table.increments('id').primary()
+      table.integer('onerep_profile_id').references('subscribers.onerep_profile_id').nullable()
+      table.integer('onerep_scan_id')
```

Comment:
Does `onerep_scan_id` need an index too?
We seem to use `where` on both `onerep_scan_id` and `onerep_profile_id` but I only see an explicit index on `onerep_profile_id` below on L18.

Sample query:
https://github.com/mozilla/blurts-server/blob/d67b9acd84c6923114327c76408687fdf8ab1730/src/db/tables/onerep_scans.js#L14-L18

Reply:
Yes I think it's appropriate here... I don't expect this table to grow at the same rate as our `subscribers` table, but it's still good practice (and I imagine that joining based on the onerep ID will be the normal use case).

We'll also use this table to keep an eye on how many scans have been performed, and how long they take, I need to think about this and test it a bit. It would be pretty easy to partition this table if it becomes performance critical (partitioning by month would be a natural fit with the constraints we have on number of scans).